src/tango/quic/tests/txn/test-transactions.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# bash strict mode
+set -euo pipefail
+IFS=$'\n\t'
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd "${SCRIPT_DIR}"
+rm -f /tmp/test-transactions.log
+NUM_TRANSACTIONS=${1:-1000}
+TX_FILE=${2:-tx}
+head -n $NUM_TRANSACTIONS $TX_FILE | while read line; do
+    echo $line
+    sudo nsenter --net=/var/run/netns/veth_test_xdp_1 ./build/native/gcc/unit-test/test_quic_txn --payload-base64-encoded $line 2>&1 | tee -a /tmp/test-transactions.log
```

Comment:
We should use `fddev txn --payload-base64-encoded` instead of `./build/native/gcc/unit-test/test_quic_txn --payload-base64-encoded`.